static/styles/left-sidebar.css

```diff
@@ -452,3 +452,7 @@ li.show-more-private-messages a {
 .zoom-in .zoom-in-hide {
     display: none;
 }
+
+.show-all-streams .icon-vector-chevron-left {
```

Comment:
@timabbott this part makes sure of that. It was working in my dev environment. Also, the only case where this rule could have been overwritten was in the file `thirdparty-fonts.css` that was moved up in [this PR ](https://github.com/zulip/zulip/pull/4358#event-1021624586)